android/build.gradle

```diff
@@ -138,5 +214,9 @@ dependencies {
     }
 
     //noinspection GradleDynamicVersion
-    implementation 'com.facebook.react:react-native:+'  // From node_modules
+    if (isNewArchitectureEnabled() && getReactNativeMinorVersion() < 71) {
```

Comment:
Do we really want to support below 0.71?

Reply:
Do you mean on the new architecture or in general?
Not my decision to make, but since you're asking I assume the answer is probably no.

Reply:
Let's not add it. Other libraries don't support New Arch where you still have to build React Native yourself either. Pinging @krizzu in case he has other opinions.

Reply:
Ok, removed in 16b21651f5566d653294f793e2bdce149931f0ab